src/Tizen.NUI/src/public/DragAndDrop/DragAndDrop.cs

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright(c) 2022 Samsung Electronics Co., Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+using System;
+using System.ComponentModel;
+using System.Runtime.InteropServices;
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+using Tizen.NUI.BaseComponents;
+
+namespace Tizen.NUI
+{
+    /// <summary>
+    /// DragAndDrop controls the drag objet and data.
+    /// </summary>
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public class DragAndDrop : BaseHandle
+    {
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public DragAndDrop() : this(Interop.DragAndDrop.New(), true)
+        {
+            if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
+        }
+
+        internal DragAndDrop(global::System.IntPtr cPtr, bool cMemoryOwn) : base(cPtr, cMemoryOwn)
+        {
+
+        }
+
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public delegate void DragAndDropEventHandler(DragEvent dragEvent);
+
+        internal delegate void InternalDragAndDropEventHandler(global::System.IntPtr dragEvent);
+
+        /// <summary>
+        /// Starts drag and drop.
+        /// </summary>
+        /// <param name="sourceView">The soruce view</param>
+        /// <param name="shadowView">The shadow view for drag object</param>
+        /// <param name="data">The data to send</param>
+        /// <returns>true if starting drag operation is success. otherwise, false.</returns>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public bool StartDragAndDrop(View sourceView, View shadowView, string data)
+        {
+            return Interop.DragAndDrop.StartDragAndDrop(SwigCPtr, View.getCPtr(sourceView), View.getCPtr(shadowView), data);
+        }
+
+        /// <summary>
+        /// Adds listener for drop targets
+        /// </summary>
+        /// <param name="targetView">The target view</param>
+        /// <param name="callback">The callback function to get drag event</param>
+        /// <returns>true if adding drop target is success. otherwise, false.</returns>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public bool AddListener(View targetView, DragAndDropEventHandler callback)
```

Comment:
C#은 에러 상황은 exception을 사용해서 알립니다.
```suggestion
        public void AddListener(View targetView, DragAndDropEventHandler callback)
```

Reply:
수정하였습니다.